src/classes/CON_Email_BATCH.cls

```diff
@@ -38,7 +38,7 @@ public class CON_Email_BATCH implements Database.Batchable<sObject>,Database.Sta
     public List<String> exception_List = new List<String>();
     public Integer recordCount = 0;
 	
-    String query = 'SELECT Id, Name, Email, Preferred_Email__c '+getCustomFields()+' FROM Contact';
+    String query = 'SELECT Id, Name, Email, Preferred_Email__c, Phone, PreferredPhone__c '+getCustomFields()+getPhoneFields()+' FROM Contact';
```

Comment:
Can we update the description at the top of the class here to say something like, "Batch Class to process contacts on EDA installation or from the EDA Settings UI via the "Run Cleanup" button. This class queries and updates all contacts to process them through both the Preferred Email and Preferred Phone functionality."